README.md

```diff
@@ -260,6 +260,9 @@ $ cargo run --release --example simple -- --host http://apache.fosciana -v -u102
 18:43:25 [ INFO] launching user 1023 from WebsiteUser...
 18:43:25 [ INFO] launching user 1024 from WebsiteUser...
 18:43:25 [ INFO] launched 1024 users...
+
+All 1024 users hatched, resetting statistics (disable with —no-reset-stats)
```

Comment:
-> `--no-reset-stats`

Reply:
Yikes, I'm not sure how that slipped in. Thanks -- I did a grep through the code-base to be sure it's fixed everywhere.